pack-mode-switcher.bat

```diff
@@ -17,12 +17,24 @@ if "%errorlevel%" == "2" goto copyExpert
 :copyNormal
 robocopy "%normalCfgPath%" "%targetPath%" *.* /e /nfl /ndl
 del "%targetPath%\globalgamerules.cfg"
```

Comment:
Can you please remove this line too, and its corresponding line in the .sh file?

Reply:
Fixed in my next commit

Reply:
Thanks!